ios/VisionCameraPluginInatVision/VisionCameraPluginInatVisionModule.m

```diff
@@ -216,8 +216,9 @@ + (VNCoreMLModel*) visionModelWithModelFile:(NSString*)modelPath {
         }
 
         // Create a new dictionary with the bestRecentBranchAsDict under the key "predictions"
+        // and the options passed in under the key "options"
```

Comment:
as @kvangork mentioned in his review of my PR, you could also return the timeElapsed to do CV for this frame as part of this response. that would allow you to report it to any logging systems we have, as well as tweak the values in options (maybe change the # of frames per second or change the threshold) if the device is struggling to keep up.

Reply:
Have added that.